plugins/Carbonix/UI/TakeoffTab.cs

```diff
@@ -165,9 +167,15 @@ private void messageBoxTimer_Tick(object sender, EventArgs e)
             // Disable the manual mode button if we are likely flying
             but_manual.Enabled = !(Host.cs.armed && (CurrentState.fromSpeedDisplayUnit(Host.cs.groundspeed) > 3 || Host.cs.ch3percent > 12));
 
+            //removed button
             // Disable final button when not at the correct waypoint
-            but_landfinal.Enabled = final_wps.Contains((int)Host.cs.wpno + 1);
+            //but_landfinal.Enabled = final_wps.Contains((int)Host.cs.wpno + 1);

```

Comment:
It's fine to delete these things instead of commenting out. That's what source control is for.